actions/release-test/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Run standard tests for release
```

Comment:
It sort of feels like we should in-line this, if it's only used by one caller and it's pretty small (and having it spelled out in the main action means the reader doesn't have to drill down into another file to see what's going on).

Reply:
Yeah, even more so than `release-publish/action.yml`